src/renderer/app.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div class="editor-container">
+  <div
+    class="editor-container"
+    :class="[{'frameless': platform !== 'darwin'}]"
```

Comment:
Code style: A space is needed after `{` and before `}`.

Reply:
:+1: